### ISSUE TITLE
I'm looking for a plugin for zigate in jeedom

### ISSUE THREAD

**Issue by handfreezer** [edgd1er/Abeille#2]
Hello,

    I'm looking for a plugin for zigate in jeedom, and I would like to know why you forked from main line plugin of Abeille?

Best regards.

**Comment by edgd1er** [comment]
Hi, 

I forked to help main dev and suggest parts by pull requests. For the moment, both projects are the same, as I'm agree with concepts applied.

**Comment by handfreezer** [comment]
Ok, so the main dev is still the one to use for simple user?

**Comment by edgd1er** [comment]
Yes, consider KiwiHC16/Abeille as the most uptodate repo for the zigBee jeedom's plugin. The doc is very well described and a google automated translation will help you to install the plugin as the jeedom market is updated for less often than github is.
There is a "how to" install in jeedom a plugin directly from github in Abeille's doc. You may already know that though.
